src/olympia/migrations/984-add-private-whiteboard.sql

```diff
@@ -0,0 +1,9 @@
+
+CREATE TABLE `private_whiteboard` (
+    `created` datetime(6) NOT NULL,
+    `modified` datetime(6) NOT NULL,
+    `addon_id` integer NOT NULL PRIMARY KEY,
+    `content` longtext NOT NULL
+) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+ALTER TABLE `private_whiteboard` ADD CONSTRAINT `addon_id_refs_id_deadbeef` FOREIGN KEY (`addon_id`) REFERENCES `addons` (`id`) ON DELETE CASCADE;
```

Comment:
How did you generate this? Django typically does not generate `ON DELETE CASCADE` statements

Reply:
By hand. How do I get django to do it?

Reply:
`./manage.py sqlall reviewers` should do it (pick the table you added + the constraints/indexes)